scene/gui/graph_node.cpp

```diff
@@ -826,6 +828,7 @@ void GraphNode::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "overlay", PROPERTY_HINT_ENUM, "Disabled,Breakpoint,Position"), "set_overlay", "get_overlay");
 
 	ADD_SIGNAL(MethodInfo("position_offset_changed"));
+	ADD_SIGNAL(MethodInfo("slot_updated"));
```

Comment:
If we are adding a new signal, why not make it universally useful? I think we should pass the index of the slot to the signal.

Reply:
Well, I used to avoid using the signal. And I still think it would be a better idea to create a new method for update connections. But I thought it's still an unsafe way.

And should I define a new method for a new signal ?

Reply:
Yes, keep `_graph_node_moved` untouched and add a new method for `slot_updated`. I think it would even be enough to only update connections layer and minimap in that case.